docs/outlook/autolaunch.md

```diff
@@ -46,7 +46,9 @@ To preview this feature:
   - The minimum required build is 16.0.14026.20000. Join the [Office Insider program](https://insider.office.com) for access to Office beta builds.
   - Configure the registry:
```

Comment:
lets explain why this is needed.  "In Outlook for Win32 we download a copy of Office.js (as opposed to load it from the CDN) . By default we download the Production version of the API. If you want to use the BETA version of the  Outlook JavaScript APIs that are in Beta you need to add this registry entry, otherwise the API will not be found. "